store/src/main/java/com/nytimes/android/external/store/base/beta/Store.java

```diff
@@ -63,12 +64,21 @@
     /**
      * Clear the memory cache of all entries
      */
+    @Deprecated
     void clearMemory();
 
     /**
      * Purge a particular entry from memory cache.
      */
+    @Deprecated
     void clearMemory(@Nonnull V key);
 
+    void clear();
+
+    /**
+     * Purge a particular entry from memory and disk cache.
+     */
+    void clear(@Nonnull V key);
```

Comment:
also mention that disk cache is only cleared if persister implements `Clearable`